docs/user/openstack/known-issues.md

```diff
@@ -81,3 +81,18 @@ The teardown playbooks provided for UPI installation will not delete:
  - Swift container for image registry (bootstrap container is correctly deleted)
 
 These objects have to be manually removed after running the teardown playbooks.
+
+## Control plane scale out anti-affinity
+
+A long-standing OpenStack Compute issue prevents the "soft-anti-affinity"
+policy to be correctly applied when instances are created in parallel. The
+Installer places the initial Control plane nodes in a Server group with
+"soft-anti-affinity" policy and creates the first three sequentially to work
+around the problem. Additional Control plane nodes beyond the first three are
+created, according to the "replicas" property, in a later phase. These nodes
+are not guaranteed to be created sequentially.
+
+When anti-affinity is a requirement, it is advised to only install with three
+nodes, and then add any further Control plane nodes by assignign them a new
```

Comment:
```suggestion
control plane nodes, and then add any further control plane nodes by assigning them a new
```

Reply:
OK, de-capitalized "Control plane" and fixed the typo